tests/container-kill_test.go

```diff
@@ -151,12 +151,25 @@ var _ = Describe("BDD of pod-delete experiment", func() {
 
 			//Checking the chaosresult
 			By("Checking the chaosresult")
-			app, err := clientSet.ChaosResults(chaosTypes.ChaosNamespace).Get(engineName+"-"+experimentName, metav1.GetOptions{})
-			Expect(string(app.Status.ExperimentStatus.Verdict)).To(Equal("Pass"), "Verdict is not pass chaosresult")
+			chaosResultVerdict, err := clientSet.ChaosResults(chaosTypes.ChaosNamespace).Get(engineName+"-"+experimentName, metav1.GetOptions{})
```

Comment:
Can you use a meaningful name, `chaosResultVerdict` this is an instance of result only so if you use `chaosResultVerdict.Status.ExperimentStatus.Verdict` then the verdict is coming twice so it is not looking good. please check the other place too

Reply:
`chaosResultVerdict`  should also be fine ... gives more meaning. cc: @chandankumar4  should I change this?

Reply:
you can use chaosResult

Reply:
only `chaosResult`?

Reply:
yes! You are getting the chaosResult only